app/components/profile_picture/image.tsx

```diff
@@ -56,7 +58,7 @@ const Image = ({author, forwardRef, iconSize, size, source, url}: Props) => {
 
         const pictureUrl = buildProfileImageUrlFromUser(serverUrl, author);
         return source ?? {uri: buildAbsoluteUrl(serverUrl, pictureUrl)};
-    }, [author, serverUrl, source]);
+    }, [author, serverUrl, source, lastPictureUpdateAt]);
```

Comment:
It's a bit concerning that this needs to be passed in separately since that sounds like `author` is being mutated for it to not cause `imgSource` to be updated

Reply:
Idk why is happening but yes I agree, i did not want to spend tons of time on this